loony_dev/web/static/app.css

```
@@ -1389,11 +1395,14 @@ button.action:hover,
 /* The screen title carries the design's .screen-title treatment (loony-dev.html):
    display size, brand --ink, bold, tight tracking. */
 .screen-head-text h2 { margin: 0; font-size: var(--fs-display); font-weight: var(--fw-bold); color: var(--ink); letter-spacing: -0.02em; line-height: 1.05; overflow-wrap: anywhere; }
-/* The restored per-screen intro copy. */
+/* The restored per-screen intro copy. Pinned to the design's `.screen-sub`
+   (loony-dev.html): 14.5px / line-height 1.5 / 7px top gap. These are literal
+   px on purpose — no --fs-*/--space-* token matches 14.5px or 7px. */
```

Stylelint (17.13.0)

[error] Expected whitespace before "*/" (comment-whitespace-inside)

(comment-whitespace-inside)